src/localization/de/learn.json

```diff
@@ -0,0 +1,163 @@
+{
+  "examples.starter.title": "Übung",
+  "examples.starter.description": "Jetzt ist es an der Zeit, etwas zu üben, um das Gelernte zu festigen.",
+
+  "examples.finish.title": "Übung abgeschlossen!",
+  "beispiele.finish.description": " ",
+
+  "steps.starter.title": "Einführung",
+  "steps.starter.description": "Regex ist die Abkürzung für `Regular Expression`. Es hilft, Text zuzuordnen, zu finden oder zu verwalten. Beginnen Sie mit der Eingabe von `OK` in das Regex-Feld, um mit dem ersten Schritt fortzufahren und auf die detailliertere Beschreibung zuzugreifen ..",
+
+  "steps.whatIsRegex.title": "Was sind reguläre Ausdrücke `Regex`?",
+  "steps.whatIsRegex.description": "Reguläre Ausdrücke sind eine Zeichenkette, die ein Suchmuster ausdrückt. Oft als `Regex` oder `Regexp` abgekürzt. Sie werden besonders verwendet, um Wörter in Texten zu finden oder zu ersetzen. Darüber hinaus können wir testen, ob ein Text den von uns festgelegten Regeln entspricht.\\n\\n Angenommen, Sie haben eine Liste mit Dateinamen und möchten nur Dateien mit der Erweiterung `pdf` finden. Geben Sie nach Eingabe eines Ausdrucks `^\\w+\\.pdf$` funktioniert. Die Bedeutung der Definitionen in diesem Ausdruck wird im Laufe der Schritte klarer.",
+
+  "steps.basicMatchers.title": "Grundlegende Matcher",
+  "steps.basicMatchers.description": "Das zu findende Zeichen oder Wort wird direkt geschrieben. Es ähnelt einem normalen Suchvorgang. Um beispielsweise das Wort `neugierig` im Text zu finden, geben Sie dasselbe ein.",
```

Comment:
`neugierig` This is the word intended to be in the text, so it should not be translated.

Expected: `curious`